catalog/content/core/content-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -45,9 +47,17 @@
 					availability="optional">
 	</reference-list>
 
+	<reference-list id="preUpdateStoragePluginList" interface="ddf.content.plugin.PreUpdateStoragePlugin"
```

Comment:
I believe making these optional actually means they can be null. In that case the calling code should cehck for that.